### DISCUSSION TITLE
Can my Gatsby Functions live inside a plugin?

### DISCUSSION THREAD

**Discussion by lanelps** [gatsbyjs/gatsby#32956] [Help] [answered]
I'm currently developing a local plugin for some e-commerce functionality. I want to move some of my lambdas into this plugin and be able to call the functions in the same way (e.g `/api/function.js`). Is this possible? How would Gatsby Cloud find/grab these functions when the site is deployed as well?

**Comment by KyleAMathews** [comment, marked as answer]
Yes indeed! https://github.com/gatsbyjs/gatsby/pull/31466

Need to update the docs still with this

**Reply by lanelps** [reply]
Awesome Kyle, thanks for the answer!
